### ISSUE TITLE
lang attribute test incorrectly fails with lang attribute present but xml:lang absent

### ISSUE THREAD

**Issue by brianhewitt** [pa11y/pa11y-dashboard#364]
For WCAG2 AA principle 3 Guideline3_1.3_1_1.H57.2:  an error is generated when the `lang=xx` attribute is present but there's no `xml:lang` attribute. The error states "The html element should have a lang **or** xml:lang attribute which describes the language of the document", but the xml:lang attribute is deprecated (see https://www.w3.org/WAI/standards-guidelines/act/rules/5b7ae0/proposed/), and it seems to be ignoring the "or" part of that statement.

Maybe this is a WCAG 2.2 change? Obviously I can ignore the rule, but I'd like to keep it to make sure there aren't pages missing the `lang` attribute.

Even if that deprecation is newer, on older standards it should suffice if `lang` OR  `xml:lang` are present (and separately, if both are present, they should be the same), correct?

<img width="689" height="246" alt="Image" src="https://github.com/user-attachments/assets/086d324c-1978-4228-8adf-6532d4622de4" />

**Comment by josebolos** [comment]
Hi @brianhewitt! Thanks for the detailed report! Unfortunately I cannot reproduce this.

This check comes from HTML_CodeSniffer, the [relevant guard for `H57.2`](https://github.com/pa11y/HTML_CodeSniffer/blob/027a6abbdea94308f37adc3eb8b44434cbba7790/Standards/WCAG2AAA/Sniffs/Principle3/Guideline3_1/3_1_1.js#L37) is:

```js
if ((element.hasAttribute('lang') === false) && (element.hasAttribute('xml:lang') === false)) {
    HTMLCS.addMessage(HTMLCS.ERROR, element, ..."3_1_1_H57.2"..., 'H57.2');
}
```

H57.2 should only fire when both lang and xml:lang are absent. Could you provide a way for us to reproduce this? Ideally a public URL that shows the error, or a minimal HTML snippet / standalone page that triggers it. That would let us confirm what pa11y is actually seeing on the <html> element and get to the bottom of it.

**Comment by brianhewitt** [comment]
Ah, my apologies. Please disregard - just a typo on my end that I overlooked a dozen times.
